<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispenseWindow</class>
 <widget class="QWidget" name="DispenseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="fillLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>640</y>
     <width>621</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:40pt; color:#ffffff;&quot;&gt;HOLD BUTTON TO FILL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fillLabelArrow">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>650</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix/ASSETS/up_arrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/assets/up_arrow.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="bottleLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>1180</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:40pt; color:#ffffff;&quot;&gt;PLACE BOTTLE HERE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="bottleLabelArrow">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>1190</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
image: url(:/new/prefix/ASSETS/side_arrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/assets/side_arrow.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="pourLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; color:#ffffff;&quot;&gt;POURING&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; color:#ffffff;&quot;&gt;TIME!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>1399</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::BottomToTop</enum>
   </property>
   <property name="format">
    <string>%vml</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>1730</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border-image: url(:/assets/V5/rect_button.png);</string>
   </property>
   <property name="text">
    <string>DONE</string>
   </property>
  </widget>
  <widget class="QLabel" name="maskLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>1920</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/assets/V6/progressbarmask.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="forwarButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
border-image: url(:/assets/rect_button.png);</string>
   </property>
   <property name="text">
    <string>FORWARD</string>
   </property>
  </widget>
  <zorder>progressBar</zorder>
  <zorder>maskLabel</zorder>
  <zorder>fillLabel</zorder>
  <zorder>bottleLabel</zorder>
  <zorder>pourLabel</zorder>
  <zorder>pushButton</zorder>
  <zorder>fillLabelArrow</zorder>
  <zorder>bottleLabelArrow</zorder>
  <zorder>forwarButton</zorder>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
